webrtc.html

```diff
@@ -3839,18 +3844,60 @@
+                  <a data-dfn-for="idl-def-rtcicecandidateinit-sdpmid">sdpMid</a></code> and <code>
+                  <a data-dfn-for="idl-def-rtcicecandidateinit-sdpmlineindex">sdpMLineIndex</a>
+                  </code> dictionary members in <var>candidateInitDict</var> are
+                  <code>null</code>, <a>throw</a> a <code>TypeError</code>.</li>
+                  <li>Let <var>iceCandidate</var> be a newly created
+                  <code>RTCIceCandidate</code> object.</li>
+                  <li>Set the corresponding attributes in <var>iceCandidate</var>
+                  with the dictionary member values of <var>candidateInitDict</var>.</li>
+                  <li>Let <var>candidate</var> be the <code>
+                  <a data-dfn-for="idl-def-rtcicecandidateinit-candidate">candidate</a></code>
+                  dictionary member of <var>candidateInitDict</var>. If
+                  <var>candidate</var> is not an empty string, run the following steps:
+                  <ol>
+                    <li>Parse <var>candidate</var> using the <a><code>candidate-attribute</code>
+                    </a> grammar.</li>
+                    <li>If parsing of <a><code>candidate-attribute</code></a> has failed, abort
+                    these steps.</li>
+                    <li>If any field in the parse result represents an invalid value for the
+                    corresponding attribute in <var>iceCandidate</var>, abort these steps.</li>
+                    <li>Set the corresponding attributes in <var>iceCandidate</var> to
+                    the field values of the parsed result.</li>
+                  </ol></li>
+                  <li>Return <var>iceCandidate</var>.</li>
+                </ol>
+                <div class="note">
+                  <p>The constructor for <code>RTCIceCandidate</code> only does basic
+                  type checking for the dictionary members in <var>candidateInitDict</var>.
+                  Detailed validation on the well-formedness of <code>candidate</code>,
+                  <code>sdpMid</code>, <code>sdpMLineIndex</code>, <code>ufrag</code>
+                  with the corresponding session description is done when passing
+                  the <code>RTCIceCandidate</code> object to <a>
+                  <code>addIceCandidate()</code></a>.</p>
+
+                  <p>To maintain backward compatibility, any error on parsing the
+                  <var>candidate</var> attribute is ignored. In such case, the
+                  <a><code>candidate</code></a> attribute holds the raw
+                  <a data-dfn-for="idl-def-rtcicecandidateinit-candidate">
+                  <code>candidate</code></a> string given in <var>candidateInitDict</var>,
+                  but derivative attributes such as <a><code>foundation</code></a>,
+                  <a><code>priority</code></a>, etc are set to their default value,
+                  which is <code>null</code>.</p>
```

Comment:
The "derivative attributes are set to `null` part" could be a bit more clear in the normative text. Maybe step 1 could be "initialize attribute values to `null`". Or is this implied?